code/game/machinery/kitchen/gibber.dm

```diff
@@ -207,8 +207,14 @@ obj/machinery/gibber/New()
 
 	var/obj/item/weapon/reagent_containers/food/snacks/meat/allmeat[totalslabs]
 	for (var/i=1 to totalslabs)
-		var/obj/item/weapon/reagent_containers/food/snacks/meat/newmeat = new occupant.meat_type(null, occupant)
-		newmeat.reagents.add_reagent (NUTRIMENT, sourcenutriment / totalslabs) // Thehehe. Fat guys go first
+		var/obj/item/weapon/newmeat		
+		if(istype(newmeat, /obj/item/weapon/reagent_containers))
```

Comment:
this check will always return null because at this point newmeat is null

Reply:
I thought I was free.

Reply:
the only species creating bones are ones without skin (plasmamen, skeletons) right?

just make this into a check like `occupant.species.anatomy_flags & NO_SKIN`

Reply:
I'd like to leave it open for future expansion.

Reply:
In that case, just make the New() of your bone sheets compatible with receiving the occupant var and it won't matter

e.g.: do way with the if/else statement, just keep `newmeat = new occupant.meat_type(null, occupant)`

Add a `/obj/item/stack/sheet/bone/New(turf/loc, mob/occupant, var/amount)` that just calls parent (or you could change its name var to `[occupant]'s bones` etc. The important part is just to ensure that you don't get runtimes from handing an unexpected mob var to a sheet object

Reply:
I considered that option when I was talking with Probe about his code @Kurfursten but personally I'm against deliberately feeding an object a parameter that it doesn't want, sanity or no sanity it just rubs me the wrong way and kinda feels like shit code